client/src/features/timeline/TimelineContainer.tsx

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React, { useRef } from 'react'
+import { Col, Row } from 'react-bootstrap'
+
+import { IHalLinks } from '../../types/IHalLinks'
+import { useGetTimelineQuery } from '../../redux/api/ml_api'
+import StyledEntityPageSection from '../../styles/shared/EntityPageSection'
+// import StyledDisplaySwitchButton from '../../styles/shared/DisplaySwitchButton'
+import {
+  sortTimelineData,
+  transformTimelineData,
+} from '../../lib/util/timelineHelper'
+
+import List from './List'
+// import Graph from './Graph'
+
+const getHalLinks = (
+  searchTags: IHalLinks,
+  providedHalLinks: any,
+): Array<string> => {
+  const filteredHalLinks: Array<string> = []
+
+  Object.keys(providedHalLinks || {}).map((link: string) =>
+    Object.keys(searchTags).map((tag) => {
+      if (
+        searchTags[tag].searchTag === link &&
+        providedHalLinks[link]._estimate > 0
+      ) {
+        filteredHalLinks.push(providedHalLinks[link].href)
+      }
+      return null
+    }),
+  )
+
+  return filteredHalLinks
+}
+
+const TimelineContainer: React.FC<{
+  searchTags: IHalLinks
+  providedHalLinks: any
+}> = ({ searchTags, providedHalLinks }) => {
+  const links = getHalLinks(searchTags, providedHalLinks)
+  // const [isFullscreen, setIsFullscreen] = useState<boolean>(false)
+  const timelineRef = useRef<HTMLDivElement>(null)
+  // const [display] = useState<'list' | 'graph'>('list')
+
+  const { data, isSuccess, isError } = useGetTimelineQuery(links)
+
+  // const setFullscreen = (): void => {
```

Comment:
can you remove the code commented out between lines 49 and 59?

Reply:
Removed.